docs/messaging.rst

```diff
@@ -1137,7 +1141,7 @@ In order to support page reloading, or a client connecting at a later stage, Jup
           'command' : 'debugInfo'
       }
 
-  Content of `debugInfo` response::
+  Content of the `debugInfo` response::
```

Comment:
I think we should get rid of the two-space indentation here, and in other places too:
```suggestion
Content of the `debugInfo` response::
```

Reply:
If you look at the generated [doc](https://jupyter-client.readthedocs.io/en/stable/messaging.html#additions-to-the-dap), this allwos to easily identify the different messages added to the spec. I think we should either keep this indentation, or add subsections.

Reply:
> add subsections

Yes maybe as a bullet list?

Reply:
Let's do this reformating part in a dedicated PR since the indentation was here before. We really need to move on and merge this one since we have more fields to add in some messages.